Common++/src/IpAddress.cpp

```diff
@@ -180,7 +178,7 @@ IPv6Address::IPv6Address(const IPv6Address& other)
 
 IPv6Address::~IPv6Address()
 {
-	delete m_pInAddr;
+	delete[] m_pInAddr;
```

Comment:
why did you add `[]` here?

Reply:
Because in constructor of IPv6Address the memory is allocated by by `new[]`. Refer to line 171.

`m_pInAddr = (in6_addr*)new uint8_t[sizeof(in6_addr)];`

The current implementation leads to memory leakage.

Reply:
When I fixed this issue, the memory consumption decreased from 1.5Gbytes downto 490 Mbytes in my application.

Reply:
Got it, that makes sense. Thanks for discovering this bug!